public/components/integrations/components/__tests__/testing_constants.ts

```diff
@@ -169,6 +169,12 @@ export const addedIntegrationData: AddedIntegrationsTableProps = {
           },
         ],
         id: 'ad7e6e30-0b99-11ee-b27c-c9863222e9bf',
+        references: [
+          {
+            dataSourceMDSId: 'f4cf2c60-2f4d-11ef-bf2d-65348e7e70ea',
+            dataSourceMDSLabel: 'dataSource',
+          },
+        ],
```

Comment:
Can we have two test constants here, one with MDS one without MDS?

Reply:
sure